domain/domain.go

```diff
@@ -338,11 +339,20 @@ func (do *Domain) loadSchemaInLoop(lease time.Duration) {
 			if err != nil {
 				log.Errorf("[ddl] reload schema in loop err %v", errors.ErrorStack(err))
 			}
-		case <-syncer.GlobalVersionCh():
+		case _, ok := <-syncer.GlobalVersionCh():
```

Comment:
How about combine `WatchGlobalSchemaVer` and `GlobalVersionCh` into one function?

```
select {
    case <- f():
}
```

`f()` will be called every time the select statement runs.